lib/vulcan.js

```diff
@@ -222,7 +222,8 @@ Vulcan.prototype = {
   isInsideTemplate: dom5.predicates.parentMatches(
       dom5.predicates.hasTagName('template')),
 
-  flatten: function flatten(tree, isMainDoc) {
+  flatten: function flatten(tree, rootDocPath) {
+    var isMainDoc = tree.href === rootDocPath;
```

Comment:
The only time this will equal is when tree.href is actually provided as the rootDocPath.  Perhaps a different way to write this is to not provide the value and test for its absence and then set it in the method like:

```
if (rootDocPath === undefined) {
  var isMainDoc = true;
  rootDocPath = tree.href;
}
```

and then call flatten with just `flatten(tree)` but I think what I have reads a little better or at least I'm ambivalent about the distinction.